Bundle/GoogleTests.mm

```diff
@@ -172,11 +178,15 @@ @implementation GoogleTestLoader
 + (void)load {
     NSBundle *bundle = [NSBundle bundleForClass:self];
     [[NSNotificationCenter defaultCenter] addObserverForName:NSBundleDidLoadNotification object:bundle queue:nil usingBlock:^(NSNotification *notification) {
-        [self registerTestClasses];
+        [GoogleTestCase initialize];
```

Comment:
Calling `+initialize` is frowned upon in ObjC; I think what you want is

    if (self == [GoogleTestCase class]) {
        [self registerTestClasses];		
    }

Does that solve it getting called twice?